code/modules/organs/organ_external.dm

```diff
@@ -1283,7 +1283,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 	amputation_point = "neck"
 //	gendered_icon = TRUE
 	encased = "skull"
-	var/list/teeth_list() = list()
```

Comment:
Same here

Reply:
What the fuck! Are teeth still in here? That's amazing.